.vscode/settings.json

```diff
@@ -1,5 +1,6 @@
 {

```

Comment:
I'd suggest to omit this for now. I need to delete the .vscode folder from the repo

Reply:
I've gone ahead and deleted it